ProcessLib/GroundwaterFlowProcess.h

```diff
@@ -329,6 +400,10 @@ class GroundwaterFlowProcess : public Process
     std::unique_ptr<typename GlobalSetup::VectorType> _rhs;
     std::unique_ptr<typename GlobalSetup::VectorType> _x;
 
+#ifdef USE_PETSC
+    std::unique_ptr<typename GlobalSetup::LinearSolver> _linearSolver;
```

Comment:
Why is it not possible to use the linar solver present in the `solve()` function above?